etl/steps/data/garden/itopf/2023-05-18/oil_spills.excluded_countries.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
If there are no excluded countries, just delete the file and don't pass `excluded_countries_path` anywhere.